source

```diff
@@ -56471,10 +56499,11 @@ interface <dfn interface>HTMLLegendElement</dfn> : <span>HTMLElement</span> {
   form owner</span> of that <span>form-associated element</span>.</p>
 
   <p>When a <span data-x="category-listed">listed</span> <span>form-associated element</span> has a
-  <code data-x="attr-fae-form">form</code> attribute and the <span>reference target</span> of the
-  <span>shadow root</span> of any <span>shadow host</span> element in the same <span>flat
-  tree</span> changes, then the user agent must <span>reset the form owner</span> of that
-  <span>form-associated element</span>.</p>
+  <code data-x="attr-fae-form">form</code> attribute and the <span>reference target</span> of any
+  <span>shadow root</span> of any <span>shadow-including inclusive ancestor</span> of the
+  <span data-x="get the attr-associated element"><code data-x="attr-fae-form">form</code>-associated
+  element</span> for the <span>form-associated element</span> changes, then the user agent must
+  <span>reset the form owner</span> of that <span>form-associated element</span>.</p>
```

Comment:
I made this change because I figure there's a more limited set of circumstances where changing a reference target can actually impact what the `form`-associated element is (that phrase just got horribly ambiguous, but hopefully you know what I mean).

Reply:
Thanks, the overloaded term made me do a double-take but this is definitely more precise.

Reply:
I wonder...does it miss anything if we instead just say this?
```html
  <p>When a <span data-x="category-listed">listed</span> <span>form-associated 
element</span>'s <span data-x="get the attr-associated element">
<code data-x="attr-fae-form">form</code>-associated element</span> changes, then
the user agent must <span>reset the form owner</span> of that
<span>form-associated element</span>.</p>
```

That would also capture the case above this that resets the form-associated element upon ID changes.

Reply:
Trying it here: https://github.com/dandclark/html/pull/5